playbooks/accounting.yml

```diff
@@ -7,5 +7,5 @@
   become: true
   roles:
     - common-server
-    - forward-server-mail
     - accounting
+    - common-server-post
```

Comment:
I at first thought Filebeat would cause issues if it found no logs as specified in the paths but it works just fine. So if we have nothing in particular to run in `-post`, we can delete that role. I'll need to rebase on your PR first though -- I'll wait for you to merge.